packages/clerk-js/src/ui/components/UserButton/SessionActions.tsx

```diff
@@ -96,7 +93,6 @@ export const MultiSessionActions = (props: SessionActionsProps) => {
           icon={SignOut}
           label={localizationKeys('userButton.action__signOut')}
           onClick={handleSignOutSessionClicked(session)}
-          variant='secondaryDanger'
```

Comment:
Also, this one needs to be red according to the designs:
![Screenshot 2024-02-02 at 12 53 22](https://github.com/clerk/javascript/assets/15199353/f610216e-e952-4b97-9e62-06527ab8c8f8)

Reply:
No it shouldn't.

Reply:
I'd assume it should be, yes, but I should also do that for the sign out all, it probably got away.